tests/functional/test_chain.py

```diff
@@ -185,3 +185,8 @@ def test_contract_caches_default_contract_type_when_used(solidity_contract_insta
     # Ensure we don't need the contract type when creating it the second time.
     contract = ape.Contract(address)
     assert isinstance(contract, ContractInstance)
+
+
+def test_set_balance(chain, test_accounts):
+    with pytest.raises(APINotImplementedError):
```

Comment:
Wait, does `eth-tester` not support this method?

Reply:
It didnt seem like it but i created an issue for them